src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ClientTrackingConfigurationTest.kt

```diff
@@ -16,7 +16,7 @@ import org.springframework.mock.web.MockHttpServletResponse
 import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.context.ContextConfiguration
 import org.springframework.test.context.junit.jupiter.SpringExtension
-import uk.gov.justice.digital.hmpps.visitscheduler.helper.JwtAuthHelper
+import uk.gov.justice.digital.hmpps.visitscheduler.test_setup.helper.JwtAuthHelper
```

Comment:
Not sure about this name, have you a better Idea? I just wanted to separate the tests from the testing config classes

Reply:
If your not sure then just leave it as it was.

Reply:
yes not sure will talk about it laters